src/app/legacy/polygon.ts

```diff
@@ -0,0 +1,53 @@
+import { Token } from '@/constants/tokens'
```

Comment:
why don't just use the tokenlists here? Or have you just released and thats why?

Reply:
yea I was lazy 😅 started out with this file but then needed the components. should have added the indices there too. will update tokenlists.